cohere/responses/custom_model.py

```diff
@@ -114,7 +135,8 @@ def from_dict(cls, data: Dict[str, Any], wait_fn) -> "BaseCustomModel":
             completed_at=_parse_date(data["completed_at"]) if "completed_at" in data else None,
             base_model=data["settings"]["baseModel"],
             model_id=data["model"]["route"] if "model" in data else None,
-            hyperparameters=HyperParameters.from_response(data["settings"]["hyperparameters"])
+            hyperparameters=HyperParameters.from_response(data["settings"]["hyperparameters"]),
+            billing=FinetuneBilling.from_response(data["billing"])
```

Comment:
What is this? Isn't finetuning free?

Reply:
we are going to start to charge finetune, the API change is ready behind feature flag